dhcpv6/dhcpv6relay.go

```diff
@@ -180,3 +181,53 @@ func (r *DHCPv6Relay) GetInnerPeerAddr() (net.IP, error) {
 	}
 	return addr, nil
 }
+
+// NewRelayReplFromRelayForw creates a RELAY_REPLY packet based on a RELAY_FORW
+// packet containing the passed DHCPv6 message as payload.
```

Comment:
just to be sure that I understand: this will create a suitable relay reply based on the relay-forward, and will replace the inner packet with what's passed as second argument?

If so, can you be a bit more explicit in this comment on the inner packet being replaced?

Reply:
That is exactly the case. I will update the comment.